spectroscopy/code_src/sdss_functions.py

```diff
@@ -39,7 +40,7 @@ def SDSS_get_spec(sample_table, search_radius_arcsec, data_release):
         try:
             xid = SDSS.query_region(search_coords, radius=search_radius_arcsec
                                     * u.arcsec, spectro=True, data_release=data_release)
-        except requests.HTTPError:
+        except (requests.HTTPError, Timeout):
```

Comment:
let's be specific and use ReadTimeout, so we still traceback if there are any issues with connection.